### PR TITLE
fix formatString describe helper

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1628789

cc @smarterclayton @soltysh

### CODE REVIEW

File: pkg/oc/lib/describe/helpers.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func formatString(out *tabwriter.Writer, label string, v interface{}) {
 
 	fmt.Fprintf(out, fmt.Sprintf("%s:", label))
 	for _, lval := range labelVals {
-		fmt.Fprintf(out, fmt.Sprintf("\t%s\n", lval))
+		fmt.Fprintln(out, fmt.Sprintf("\t%s", lval))
```
[the remark this answers is not in the document]
Comment (juanvallejo) [reply pkg/oc/lib/describe/helpers.go]: >  The current won't work, b/c fmt.Fprintln does not accept format parameter

@soltysh I'd say that's precisely why `Fprintln` fixes the issue :) The problem was that label values containing `%` were being incorrectly interpreted as fmt strings. From the bugzilla:

```
Log Tail:	Pulled 2/2 layers, 100%!c(MISSING)omplete
		Extracting

		Pulling image golang:1.10.3 ...
		error: build error: no such image
```

[the remark this answers is not in the document]
Comment (soltysh) [reply pkg/oc/lib/describe/helpers.go]: My bad, I was parsing this line wrongly.

**Pull request merged**